pyproject.toml

```diff
@@ -1,140 +1,31 @@
 [build-system]
-requires = [
-    "hatchling>=1.8.1",
-    "jupyter_core",
-]
 build-backend = "hatchling.build"
+requires = ["hatchling>=1.5.0,<2"]
 
 [project]
-name = "voici"
-description = "Voici turns Jupyter notebooks into static web applications"
+name = "voici-root"
 readme = "README.md"
-requires-python = ">=3.8"
-authors = [
-    { name = "Voila Development Team" },
-]
-keywords = [
-    "Jupyter",
-    "JupyterLite",
-    "Emscripten",
-    "Voila",
-    "Voici",
-]
-classifiers = [
-    "Framework :: Jupyter",
-    "License :: OSI Approved :: BSD License",
-    "Programming Language :: Python",
-    "Programming Language :: Python :: 3",
-    "Programming Language :: Python :: 3.8",
-    "Programming Language :: Python :: 3.9",
-    "Programming Language :: Python :: 3.10",
-    "Programming Language :: Python :: 3.11",
-]
-dependencies = [
-    "voila>=0.5.5,<0.6",
-    "jupyterlite-core>=0.2.0,<0.3.0",
-]
-dynamic = [
-    "version",
-]
-
-[project.license]
-file = "LICENSE"
-
-[project.optional-dependencies]
-dev = [
-    "black",
-    "ruff==0.0.258",
-    "hatch",
-]
-
-docs = [
-    "myst-parser",
-    "pydata-sphinx-theme",
-    "sphinx",
-    "sphinx-autobuild",
-    "sphinxcontrib-video",
-]
-
-[project.scripts]
-voici = "voici.app:main"
-
-[project.entry-points."jupyterlite.addon.v0"]
-voici = "voici.addon:VoiciAddon"
-
-[project.urls]
-Homepage = "https://github.com/voila-dashboards/voici"
+license = { file = "LICENSE" }
+dynamic = ["version"]
 
 [tool.hatch.version]
-path = "voici/_version.py"
-
-[tool.hatch.build]
-artifacts = [
-    "voici/static/build",
-]
-
-[tool.hatch.build.targets.sdist]
-exclude = [
-    ".github",
-    "*.gif",
-]
-
-[tool.hatch.build.hooks.jupyter-builder]
-dependencies = [
-    "hatch-jupyter-builder>=0.5.3",
-]
-build-function = "hatch_jupyter_builder.npm_builder"
-ensured-targets = [
-    "voici/static/build/voici.js",
-]
-skip-if-exists = [
-    "voici/static/build/voici.js",
-]
-
-[tool.hatch.build.hooks.jupyter-builder.editable-build-kwargs]
-build_cmd = "build"
-npm = [
-    "yarn",
-]
-force = true
-
-[tool.hatch.build.hooks.jupyter-builder.build-kwargs]
-build_cmd = "build:prod"
-npm = [
-    "yarn",
-]
+path = "python/voici/voici/_version.py"
 
 [tool.jupyter-releaser]
-skip = [
-    "check-links",
+skip = [ "check-python" ]
```

Comment:
I guess we're facing this issue if `voici` depends on `voici-core`: https://github.com/jupyter-server/jupyter_releaser/issues/499

Reply:
Currently `check-python` is also disabled in the JupyterLite repo.

Reply:
Ok thanks!

Reply:
Can you elaborate on why "build-python" is also disabled on the root of the jupyterlite repo?

Reply:
Ah likely because the jupyterlite repo does not uses the `python_packages` config of the releases, and still relies on `doit build` for building the python packages. This might get simplified with https://github.com/jupyterlite/jupyterlite/issues/684 hopefully.

Reply:
Even with `python_packages` config, jupyter releaser still has issue with python check on releasing the packages.

Reply:
> Even with `python_packages` config, jupyter releaser still has issue with python check on releasing the packages.

Yes still https://github.com/jupyter-server/jupyter_releaser/issues/499